be/src/storage/page_cache.cpp

```diff
@@ -51,6 +51,11 @@ StoragePageCache::StoragePageCache(MemTracker* mem_tracker, size_t capacity)
 StoragePageCache::~StoragePageCache() {}
 
 void StoragePageCache::set_capacity(size_t capacity) {
+#ifndef BE_TEST
```

Comment:
Why need this macro ?

Reply:
> Why need this macro ?

This is the same behavior as before, look at this.
`
void StoragePageCache::insert(const CacheKey& key, const Slice& data, PageCacheHandle* handle, bool in_memory) {
#ifndef BE_TEST
    int64_t mem_size = malloc_usable_size(data.data);
    tls_thread_status.mem_release(mem_size);
    MemTracker* prev_tracker = tls_thread_status.set_mem_tracker(_mem_tracker);
    tls_thread_status.mem_consume(mem_size);
    DeferOp op([&] { tls_thread_status.set_mem_tracker(prev_tracker); });
#endif

    auto deleter = [](const starrocks::CacheKey& key, void* value) { delete[](uint8_t*) value; };

    CachePriority priority = CachePriority::NORMAL;
    if (in_memory) {
        priority = CachePriority::DURABLE;
    }

    auto* lru_handle = _cache->insert(key.encode(), data.data, data.size, deleter, priority);
    *handle = PageCacheHandle(_cache.get(), lru_handle);
}
`
Besides some test cases may not require MemTracker.